### PR TITLE
Added missing spec files for API800/1000/1200 for all network resources

### PR DESCRIPTION
**Pull request by yuvirani**
### Description
Added missing spec files for API800/1000/1200 for all network resources

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass (`$ rake test`).
- [x] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] New endpoints supported are updated in the endpoints-support.md file.
- [x] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: spec/unit/resource/api1000/synergy/network_set_spec.rb [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+require 'spec_helper'
+
+RSpec.describe OneviewSDK::API1000::Synergy::NetworkSet do
+  include_context 'shared context'
+
+  it 'inherits from OneviewSDK::API800::C7000::NetworkSet' do
+    expect(described_class).to be < OneviewSDK::API800::C7000::NetworkSet
```
Comment (chebroluharika) [inline spec/unit/resource/api1000/synergy/network_set_spec.rb:18]: It should inherit from API1000::C7000. Please change description accordingly.

File: spec/unit/resource/api1200/c7000/network_set_spec.rb [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+RSpec.describe OneviewSDK::API1200::C7000::NetworkSet do
+  include_context 'shared context'
+
+  it 'inherits from OneviewSDK::API1000::C7000::NetworkSet' do
+    expect(described_class).to be < OneviewSDK::API1000::C7000::NetworkSet
+  end
+
+  describe '#initialize' do
+    context 'OneView 4.00' do
```
Comment (chebroluharika) [inline spec/unit/resource/api1200/c7000/network_set_spec.rb:22]: It should be Oneview 5.00

File: spec/unit/resource/api1200/synergy/network_set_spec.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+RSpec.describe OneviewSDK::API1200::Synergy::NetworkSet do
+  include_context 'shared context'
+
+  it 'inherits from OneviewSDK::API1000::C7000::NetworkSet' do
+    expect(described_class).to be < OneviewSDK::API1000::C7000::NetworkSet
+  end
+
+  describe '#initialize' do
+    context 'OneView 4.00' do
```
Comment (chebroluharika) [inline spec/unit/resource/api1200/synergy/network_set_spec.rb:22]: Please change this to Oneview 5.00

File: spec/unit/resource/api800/synergy/network_set_spec.rb [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+require 'spec_helper'
+
+RSpec.describe OneviewSDK::API800::Synergy::NetworkSet do
+  include_context 'shared context'
+
+  it 'inherits from OneviewSDK::API600::C7000::NetworkSet' do
+    expect(described_class).to be < OneviewSDK::API600::C7000::NetworkSet
```
Comment (chebroluharika) [inline spec/unit/resource/api800/synergy/network_set_spec.rb:18]: It should inherit from API800::C7000

File: spec/unit/resource/api1000/synergy/network_set_spec.rb (hunk 1 above)
Comment (yuvirani) [reply spec/unit/resource/api1000/synergy/network_set_spec.rb]: Updated all the changes mentioned in this PR

**Pull request merged**